doc-changelog/action.yml

```diff
@@ -266,6 +266,6 @@ runs:
         if [ ! -z "$modified" ]; then
           echo "modified: $modified"
           # Commit and push fragment
-          git commit -m "chore: adding changelog file $fragment"
+          git commit -m "chore: adding changelog file $fragment [dependabot-skip]"
```

Comment:
Doesn't it have to go at the beginning? Maybe I'm confused

Reply:
I think it can be anywhere, but I can move it to the front if it looks better

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#allowing-dependabot-to-rebase-and-force-push-over-extra-commits

Reply:
No worries, let's leave it there :)